earn/src/data/Uniboost.ts

```diff
@@ -36,6 +36,8 @@ export class BoostCardInfo {
     public readonly currentTick: number,
     public readonly token0: Token,
     public readonly token1: Token,
+    public readonly lender0: Address,
+    public readonly lender1: Address,
```

Comment:
By adding these to the class, are you generating extra re-renders? i.e., all graphs are rendering as before and liquidation thresholds are calculated as before... but then they're rendered/calculated _again_ once `lender0` and `lender1` come in?

Reply:
If this is the case, we should track lender0 and lender1 in some other data type.

Reply:
Good catch

Reply:
Actually, the way we currently do things, there should be no additional chart renders as we have a guard clause that ensures marketData is fetched before constructing the BoostCardInfos.